Lib/fontTools/ttLib/tables/otBase.py

```diff
@@ -596,13 +600,16 @@ def __getattr__(self, attr):
 
 		raise AttributeError(attr)
 
-	def ensureDecompiled(self):
+	def ensureDecompiled(self, recurse=False):
```

Comment:
Doesn't recurse False default mean that by default ensureDecompiled does not actually ensure decompilation?

Reply:
`BaseTable.ensureDecompiled` pre-existed this PR, it's used in a few places. I don't want to change its default behavior. When calling `font["GSUB"].ensureDecompiled` method you are not calling this method directly, but through the `BaseTTXConverter.ensureDecompiled` (which I added) and the latter calls `BaseTable.ensureDecompiled(recurse=True)`